e2e/testcafe-devextreme/tests/chat/messageList.ts

```diff
@@ -145,3 +146,45 @@ test('Messagelist should scrolled to the latest messages after being rendered in
     }],
   });
 });
+
+test('Messagelist with date headers', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'Messagelist with date headers.png', { element: '#container' });
```

Comment:
Why didn't we demonstrate the 'Today' and 'Yesterday' configuration in this screenshot?

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925